src/main/scala/com/campudus/tableaux/TableauxConfig.scala

```diff
@@ -7,7 +7,10 @@ import io.vertx.scala.ScalaVerticle
 import scala.reflect.io.Path
 
 object TableauxConfig {
-  def apply(verticle: ScalaVerticle, databaseConfig: JsonObject, workingDir: String, uploadsDir: String): TableauxConfig = {
+  def apply(verticle: ScalaVerticle,
+            databaseConfig: JsonObject,
+            workingDir: String,
+            uploadsDir: String): TableauxConfig = {
```

Comment:
This would look nicer:
```scala
def apply(
  verticle: ScalaVerticle,
  databaseConfig: JsonObject,
  workingDir: String,
  uploadsDir: String
): TableauxConfig = {
  ...
```